### PR TITLE
Fix typo.

### PR DESCRIPTION
**Pull request by stbof**

### CODE REVIEW

File: StyleGuide.md [hunk 1]
```diff
@@ -736,7 +736,7 @@ For some examples, see [Connect to a CockroachDB Cluster](https://www.cockroachl
 
 #### Escaping special characters
```
Comment (stbof) [inline StyleGuide.md:737]:
```suggestion
#### Escape special characters
```

Comment (stbof) [reply StyleGuide.md:737]: Use imperative instead of gerund.

**Review by andf-crl — COMMENTED** [review]
Thank you for `achieve`.

However:

Your heading change breaks two links and changes a consideration into an always-true guidance, which it must not be. Escaping special characters in, say, a monospace link would render the Unicode as-is ([example](https://github.com/cockroachdb/docs/blob/9e1582ebf1f5bd662ef966db8dcb81bdaa621e69/v22.1/alter-role.md?plain=1#L41-L42)). I recommend we keep the header as is: the phrasing of the two now-broken links suggests the use cases where this heading is appropriate, as does the use of the word `may` just underneath.

**Comment by andf-crl** [comment]
"How to" sound good, thanks!

**Comment by stbof** [comment]
> "How to" sound good, thanks!

Do you want me to update the heading and fix the links?

**Comment by andf-crl** [comment]
Either: as-proposed or as-is is sufficient for me! Happy to approve either choice you make.

**Comment by stbof** [comment]
> Either: as-proposed or as-is is sufficient for me! Happy to approve either choice you make.

Done.

**Review by andf-crl — APPROVED** [review]
LGTM, thank you!

**Pull request merged**